packages/react/src/lib/accessibility/Behaviors/List/selectableListBehavior.ts

```diff
@@ -8,8 +8,10 @@ import { ListBehaviorProps } from './listBehavior'
  *
  * @specification
  * Adds role='listbox'.
- * Triggers 'moveNext' action with 'ArrowDown' on 'root'.
- * Triggers 'movePrevious' action with 'ArrowUp' on 'root'.
+ * Triggers 'moveNext' action with 'ArrowDown' on 'root', when orientation is vertical.
+ * Triggers 'moveNext' action with 'ArrowRight' on 'root', when orientation is horizontal.
+ * Triggers 'movePrevious' action with 'ArrowUp' on 'root', when orientation is vertical.
+ * Triggers 'movePrevious' action with 'ArrowLeft' on 'root', when orientation is horizontal.
  * Triggers 'moveFirst' action with 'Home' on 'root'.
  * Triggers 'moveLast' action with 'End' on 'root'.
  */
```

Comment:
Please add "aria-orientation" attribute(on the root element, under line 19), based on the aria:
https://www.w3.org/TR/wai-aria-practices/#Listbox
If options are arranged horizontally, the element with role listbox has aria-orientation set to horizontal. The default value of aria-orientation for listbox is vertical.

Thank you :)

Reply:
Added, please take a look and propose how we can test it :\